### ISSUE TITLE
Detox no longer builds with 2.10.0

### ISSUE THREAD

**Issue by scottwio** [software-mansion/react-native-reanimated#3556]
### Description

When using the latest version of reanimated 2.10.0 I can no longer create detox test builds. When running `assembleAndroidTest` the task fails with the error message.
```
Execution failed for task ':react-native-reanimated:mergeDebugAndroidTestJavaResource'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.MergeJavaResWorkAction
   > 23 files found with path 'META-INF/MANIFEST.MF' from inputs:
```
I can fix this by reverting to 2.9.1. When back on 2.9.1 the build completes without issue. It looks like the packaging options have changed between two versions, might have something to do with it.

### Steps to reproduce

1. Create a new react native project on 0.68.2
2. Follow detox sets for setting up react native do not use `precompiled .aar`
3. try and run `assembleAndroidTest`

### Reanimated version

2.10.0

### React Native version

0.68.2

### Platforms

Android

### JavaScript runtime

Hermes

### Workflow

React Native (without Expo)

### Architecture

Paper (Old Architecture)

### Build type

Debug mode

### Device

Android emulator

### Device model

Emulator system-images;android-28;default;arm64-v8a

### Acknowledgements

Yes

**Comment by scottwio** [comment]
@tomekzaw perfect that does the job, thanks.

**Comment by olexanderPotebenko** [comment]
@tomekzaw Please advise how I can accomplish this #3384 by making changes to the build.gradle?

**Comment by tomekzaw** [comment]
Hey @olexanderPotebenko, you still need to edit `node_modules/react-native-reanimated/android/build.gradle` to apply the fix from #3384. You can achieve this either with [patch-package](https://www.npmjs.com/package/patch-package) or with a custom bash script.
